server/form-pages/assess/matchingInformation/matchingInformationTask/matchingInformation.test.ts

```diff
@@ -0,0 +1,191 @@
+import { itShouldHaveNextValue, itShouldHavePreviousValue } from '../../../shared-examples'
+
+import MatchingInformation from './matchingInformation'
+
+describe('MatchingInformation', () => {
+  describe('title', () => {
+    expect(
+      new MatchingInformation({
+        apGender: 'female',
+        apType: 'esap',
+        wheelchairAccessible: 'essential',
+        singleRoom: 'essential',
+        adaptedForHearingImpairments: 'essential',
+        adaptedForVisualImpairments: 'essential',
+        adaptedForRestrictedMobility: 'essential',
+        cateringRequired: 'essential',
+        contactSexualOffencesAgainstAnAdultAdults: 'relevant',
+        nonContactSexualOffencesAgainstAnAdultAdults: 'relevant',
+        contactSexualOffencesAgainstChildren: 'relevant',
+        nonContactSexualOffencesAgainstChildren: 'relevant',
+        nonSexualOffencesAgainstChildren: 'relevant',
+        arsonOffences: 'relevant',
+        hateBasedOffences: 'relevant',
+        vulnerableToExploitation: 'relevant',
```

Comment:
Is it worth setting this as a variable at the top of the tests, as the constuctor is so meaty?

Reply:
A wise move!